src/app/routes/_message.router.js

```diff
@@ -115,6 +118,19 @@ export default (app, router, io) => {
         current: populated
       });
 
+      // notify possible mentions
+      const mentions = messageText.match(/@\w+/g);
+
+      if (mentions) {
+        mentions.map(mention => push.send(mention.substring(1), {
```

Comment:
Then the result of Mongoose's `.find()` (here https://github.com/Sirimangalo/meditation-plus-server/blob/feat/mention/src/app/helper/push.js#L10) is empty and nothing happens.